pom.xml

```diff
@@ -212,6 +212,7 @@
             <parallel>classes</parallel>
             <perCoreThreadCount>true</perCoreThreadCount>
             <threadCount>1</threadCount>
+          	<forkCount>1.5C</forkCount>
```

Comment:
```suggestion
          	<forkCount>1C</forkCount>
```

Reply:
Seems like those randomly failing tests have been improved a bit to enable more concurrency. 1.5C seemed to be too much for now, but 1C should work.